onediff_comfy_nodes/modules/booster_cache.py

```diff
@@ -50,9 +50,14 @@ def _(model: ModelPatcher):
 
 @get_cached_model.register
 def _(model: VAE):
+    if is_oneflow_available() and not is_disable_oneflow_backend():
+        from .oneflow.utils.booster_utils import is_using_oneflow_backend
+
+        if is_using_oneflow_backend(model):
```

Comment:
There is an exception thrown from is_using_onerlow_backend that is never caught:

```
  File "/src/comfyui_custom_nodes/onediff_comfy_nodes/modules/booster_cache.py", line 58, in _
    if is_using_oneflow_backend(model):
  File "/src/comfyui_custom_nodes/onediff_comfy_nodes/modules/oneflow/utils/booster_utils.py", line 86, in is_using_oneflow_backend
    raise RuntimeError("")